src/dso_api/dynamic_api/permissions.py

```diff
@@ -224,7 +230,7 @@ def _parse_filter(v: str) -> tuple[str, str]:
         field_name = v
         op = "exact"
     elif not v.endswith("]"):
-        raise ValidationError(f"missing closing bracket (]) in {v!r}")
+        raise FilterSyntaxError(f"missing closing bracket (]) in {v!r}")
```

Comment:
What happens when an opening bracket is missing. That is alread caught I suppose?

Reply:
That produces a 400 with a `not found in schema` message. I can add a case for it to deal with it more cleanly.

Reply:
Whatever you wish. To me this is already OK.

Reply:
Added a check and tests. Please review again.